src/main/java/de/muenchen/mobidam/s3/S3RouteBuilder.java

```diff
@@ -33,15 +34,25 @@ public void configure() {
                 .handled(true)
                 .process(exchange -> {
                     var s3Exception = exchange.getProperty(Exchange.EXCEPTION_CAUGHT, S3Exception.class);
-                    log.error("Error occurred in route", s3Exception);
+                    if (exchange.getIn().getHeader(Constants.CAMEL_SERVLET_CONTEXT_PATH).equals(Constants.ARCHIVE_ENTITY)) {
```

Comment:
Das kann man in eine Methode "logException" auslagern.

Reply:
✅